spec/collection_schema.yaml

```diff
@@ -0,0 +1,13 @@
+name: collection_schema
+type: object
+required: ['name', 'type', 'schema']
+properties:
+  name:
```

Comment:
There are also these fields:
* delta (boolean) -- whether this collection uses "time travel" features (has "id", "from", "to" fields)
* indexes (array of objects with properties for "type" (string/enum) and "fields" (array of string))

Reply:
I would also add `additionalProperties: false` here